src/Codeception/Module/Facebook.php

```diff
@@ -265,19 +262,19 @@ public function postToFacebookAsTestUser($params)
      *
      * Please, note that you must have publish_actions permission to be able to publish to user's feed.
      *
-     * @param string $placeId Place identifier to be verified against user published posts
+     * @param string $message published post to be verified against the actual post on facebook
      */
-    public function seePostOnFacebookWithAttachedPlace($placeId)
+    public function seePostOnFacebookWithAttachedPlace($message)
     {
         $posts = $this->facebook->getLastPostsForTestUser($this->grabFacebookTestUserAccessToken());
         if ($posts['data']) {
             foreach ($posts['data'] as $post) {
-                if (array_key_exists('place', $post) && ($post['place']['id'] == $placeId)) {
+                if (array_key_exists('message', $post) && ($post['message'] == $message)) {
                     return; // success
                 }
             }
         }
 
-        $this->fail('Failed to see post on Facebook with attached place with id ' . $placeId);
+        $this->fail('Failed to see post on Faceboo: ' . $message);
```

Comment:
a half of message is missing.
this assertion would generate a nicer result if it used some assertion for passing.